File: level68/classwork/codwars1.py
#https://www.codewars.com/kata/517abf86da9663f1d2000003/train/python
def to_camel_case(text):
    words = text.replace('_', '-').split('-')
    return words[0] + ''.join([x.title() for x in words[1:]])

File: level 072/classwork/codwars1.js

//https://www.codewars.com/kata/581e014b55f2c52bb00000f8/train/javascript
function decipherThis(text) {
  let words = text.split(' ')
  let result = []

  for (let word of words) {
    let num = ''
    let i = 0
    while (i < word.length && '0123456789'.includes(word[i])) {
      num += word[i]
      i++
    }
    let firstChar = String.fromCharCode(Number(num))
    let rest = word.slice(i).split('')
    if (rest.length > 1) {
      let temp = rest[0]
      rest[0] = rest[rest.length - 1]
      rest[rest.length - 1] = temp
    }
    result.push(firstChar + rest.join(''))
  }

  return result.join(' ')
}